GitLab support GA (#7146)
circleci/circleci-docs: jekyll/_cci2/gitlab-integration.adoc

[#overview]
== Overview

> This document walks you through integrating a GitLab project with CircleCI. It introduces you to new concepts and ways to manage your CI/CD pipelines. In this document, we also outline what we are working on for future releases.

Reviewer: ```suggestion
This document walks you through integrating a GitLab project with CircleCI. The sections below will introduce you to new concepts and ways to manage your CI/CD pipelines. We also outline what we are working on for future releases.
```
